### DISCUSSION TITLE
On click joint beta to import my database : There was an error contacting the server. Please try again.

### DISCUSSION THREAD

**Discussion by freddneos** [planetscale/discussion#193] [Bug Reports] [answered]
Hello Guys ✋ 

I'm 2 days trying to start the planetscale PoC and I'm facing this issue ...
The request response is 404 (network tab on chrome)...

On click joint beta to import my database -> There was an error contacting the server. Please try again.
<img width="1167" alt="image" src="https://user-images.githubusercontent.com/36821426/168575881-83523137-e6e6-4294-8b75-9b3ef54fb2c9.png">

Any of you facing this issue ? 
I must to be Pro Account to use this feature ?

Please any help will be welcome. Have a nice day!

**Comment by fooforge** [comment, marked as answer]
@freddneos Oops, I can reproduce this. We'll take a look shortly and I'll give you another update once it's been fixed.

**Reply by jgreet** [reply]
@freddneos Sorry about that. We fixed an issue in the web app, you should be able to join the beta now!
